virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar-icon/launchbar-icon.component.ts

```diff
@@ -85,16 +87,28 @@ export class LaunchbarIconComponent {
     this.launchbarItem.showIconLabel = false;
   }
 
+  onMouseLeaveInstanceView(event: MouseEvent) {
+    if (!this.mouseDownListener) {
+      window.addEventListener('mousedown', () => this.onMouseDownInstanceView());
+      this.mouseDownListener = true;
+    }
+  }
+
   onMouseEnterInstanceView(event: MouseEvent) {
     this.launchbarItem.showIconLabel = false;
+    if (!this.mouseDownListener) {
+      window.addEventListener('mousedown', () => this.onMouseDownInstanceView());
+      this.mouseDownListener = true;
+    }
   }
 
-  @HostListener('document:mousedown', ['$event'])
-  onMouseDownInstanceView(event: MouseEvent) {
+  onMouseDownInstanceView() {
```

Comment:
HostListener is Angular and can't be destroyed manually, so it's not good to use here